modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/Modernizer.java

```diff
@@ -170,6 +176,9 @@ public void visit(int version, int access, String name, String signature,
     public MethodVisitor visitMethod(int access, final String methodName,
             final String methodDescriptor, final String methodSignature,
             String[] exceptions) {
+        if (ignoreMethod(methodName, methodDescriptor)) {
+            return null;
+        }
```

Comment:
It seems like the other exclusions are checked inside of the `checkToken` method, should this be moved there?

Reply:
`checkToken` is called on each token inside a method. If we move this to `checkToken` we'll end up visiting each token in every method but ignoring them (if present in a method to be ignored) at the last step. That's the reason I thought of placing it here.

Ignoring classes is done in a similar way in `visit` method of `ModernizerClassVisitor`. Also, there's a second check to ignore a class in `checkToken`.